examples/paper/03-dependent-binding.makam

```diff
@@ -1,90 +1,293 @@
 %use "02-binding-forms".
 
-(* Sneak peek: work-in-progress and not much commentary here, just roughly showing what's possible. *)
+(* The type system of λProlog can be viewed as a particular subset of System Fω: namely, it is the
+   simply-typed lambda calculus extended with prenex polymorphism and simple type constructors of
```

Comment:
"Simply-typed" should be "simply typed": no hyphens after "-ly" adverbs in English (though almost everyone in the PL world ignores the rule, I learned it from the editor for my book!).